scripts/scratch_ingest/loom_ingest.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import loompy
+import sys
+import os
+import  pyrebase
+from google.cloud import firestore
+from itertools import islice
+import time
+
+db = firestore.Client()
+
+np.set_printoptions(precision=8, threshold=sys.maxsize, edgeitems=1e9)
+expression_dictionaries= dict()
+loom_file = sys.argv[1]
```

Comment:
I'd suggest using `argparse` here; it's the convention used elsewhere in this repo for parsing command-line arguments in Python.  It enables useful, standard usage documentation from e.g. `loom_ingest.py --help`.   Example: https://github.com/broadinstitute/single_cell_portal/blob/master/scripts/manage_study.py#L71.